matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -1124,10 +1124,10 @@ var PhenoTips = (function (PhenoTips) {
         return table;
     },
 
-    _generateRecipientCell : function(info, type)
+    _generateRecipientCell : function(info, type, isFrom)
     {
-        // notifications should have something generic e.g. "PhenomeCentral" (no link)
-        if (type == "notification") {
+        // The `From` field should be " PhenomeCentral" for all `notification`s, regardless of which admin sent them
+        if (type == "notification" && isFrom) {
```

Comment:
For better code readability, I would actually move the `if` up to line 1104, and only pass 1 parameter `info` to this function.

Reply:
Another comment about lines 1136-1145 below: the code for obtaining the URL is brittle. - please generate the URL in the backend and include it in the JSON with the rest of the user info.

Reply:
@marta- should we also change it [here](https://github.com/phenotips/phenotips/blob/master/components/patient-access-rules/ui/src/main/resources/PhenoTips/AccessRightsManagement.xml#L1013)?

Reply:
^ Yes, please create an issue for it.

Reply:
Created [PT-3942](https://phenotips.atlassian.net/browse/PT-3942) for it.